README.md

```diff
@@ -133,3 +133,7 @@ Given that the instructions in the [download section](#download) have been follo
 ```
 where `<keypair_name>.sec.pem` the private key created in the [relevant section](#create-keys) and `<file_to_decrypt>` one of the files downloaded following the instructions of the [download section](#download-file).
 
+
+# Create new release
+
+The github actions include a release workflow that builds binaries for different operating systems. In order to create a new release, create a tag either using the graphical interface or through the command line. That should trigger the creation of a release with the latest code of the specified branch.
```

Comment:
If I understand the `release.yml` correctly, releases are only created on tags that match `v*` right? That should probably be metioned here if that's the case :)